test/e2e/integreatly_test.go

```diff
@@ -100,6 +101,12 @@ func TestIntegreatly(t *testing.T) {
 		t.Run("Cluster", func(t *testing.T) {
 			IntegreatlyCluster(t, f, ctx)
 		})
+
+		for _, test := range common.AFTER_INSTALL_TESTS {
```

Comment:
If we have to create a new test list this must be also part of the functional suite